### DISCUSSION TITLE
Is there a way to make a message box in this theme?

### DISCUSSION THREAD

**Discussion by zSynctic** [rdbende/Azure-ttk-theme#34] [Question] [answered]
I want to know if there is a way to create a message box in this style because I use the theme in everything except the message box so when the message box appears the program appears without ttk but then when I press ok at the message box it comes back to ttk

**Comment by rdbende** [comment, marked as answer]
I assume you want to make it for Windows or Mac, since on Linux every messagebox follows the ttk theme.
You can create your own messagebox widget, or [here is](https://github.com/rdbende/Sun-Valley-messageboxes) what I made for my [Sun Valley](https://github.com/rdbende/Sun-Valley-ttk-theme) theme.
